api/src/apps/apps.rs

```diff
@@ -627,9 +452,7 @@ mod tests {
             &service_configs!("service-a"),
         )?;
 
-        let info = RequestInfo::new(Url::parse("http://example.com").unwrap());
-
-        let deployed_apps = apps.get_apps(&info)?;
+        let deployed_apps = apps.get_apps()?;
```

Comment:
Hier scheint Codeduplizierung vorgelegen zu haben, zumindest musste die gleiche Änderung mehrmals gemacht werden.
Kann man ggf noch ein wenig Duplizierung reduzieren?

Reply:
These are the unit tests and this is the part for testing, so no duplication in my opinion. here just the method's parameters where changed and so no requestInfo is needed anymore.